onadata/libs/utils/async_status.py

```diff
@@ -3,14 +3,18 @@
 PENDING = 0
 SUCCESSFUL = 1
 FAILED = 2
+PROGRESS = 3
+RETRY = 4
+STARTED = 5
```

Comment:
Between `PENDING`? `PENDING` is a unkown state, I have added `PROGRESS` to indicate that the task has started and is processing.
Docs: http://docs.celeryproject.org/en/latest/reference/celery.states.html#all-states